builtin/providers/aws/resource_aws_cloudwatch_event_rule.go

```diff
@@ -6,11 +6,12 @@ import (
 	"regexp"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
 
```

Comment:
Looks like we have some import formatting issues here - I realise they were there before but could you remove the whitespace and run `goimports` on this since we have related changes?